specification/network/resource-manager/Microsoft.Network/2017-09-01/examples/ApplicationSecurityGroupCreate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2017-09-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "applicationSecurityGroupName": "test-asg",
+    "location": "westus",
+    "parameters": {
+      "properties": {
```

Comment:
why have empty properties object. location is not a path parameter like "resourceGroupName". It is a property of the body parameter. Hence `"parameters": { "location": "westus" }`. Please be alert while providing examples. These examples would end up on the REST API docs that customers would refer to.

Reply:
"properties" should be empty, but "location" should be in the body (part of Resource object at the same level as "properties", not in the URL. Fixed.